Lib/test/support/__init__.py

```diff
@@ -2118,3 +2119,19 @@ def clear_ignored_deprecations(*tokens: object) -> None:
     if warnings.filters != new_filters:
         warnings.filters[:] = new_filters
         warnings._filters_mutated()
+
+
+# This number should be CPython internal "hot" count times 2.
+CPYTHON_WARMUP_COUNT = 8*2
+
+
+def cpython_warmup(f):
+    """Makes a test "hot" so that PEP 659 adaptive interpreter opcodes will
+    execute instead of normal opcodes.
```

Comment:
I suggest to explain that it runs a test multiple times, first with non-adaptative opcode, and then again with adaptative opcode, to test both opcodes and code paths. You may mention bpo-46465 and CHECK_EVAL_BREAKER() to justify the usage of this decorator.